SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc

```diff
@@ -13,6 +13,9 @@
 
 [sdpi_transaction#transaction_dev_23,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_23_sequence,messages='message_announce_network_presence']
 
+##NOTE THAT THE FIRST SENTENCE SHOULD BE THE SUMMARY THAT IS PUBLISHED IN THE TF-0 APPENDIX B TRANSACTIONS TABLE - but substituting with first sentence and not reftext ... HOW???
+##
+[[transaction_summary_dev_23,DEV-23 Announce Network Presence]]

```

Comment:
What kind of markup is that?

Reply:
This ## ... ##  was inserted by IntelliJ when I wanted to highlight (yellow) a section of a block.  Note that in branch 117, esp. Supplement & Volume 0 stuff, I integrated highlighted comments for items that either I wanted to make sure we discussed, or that I didn't have a clue HOW to do something (more likely the case!)  

IF the question is around the insertion of the transaction_summary_ block anchor, we can talk about the best way of formalizing AND where to formalize the summary language, which is in the TF-0 Appendix B file